src/values/TimeValue.js

```diff
@@ -294,13 +294,13 @@ SELF.getCalendarModelTextByUri = function( uri ) {
  * @since 0.7
  */
 SELF.PRECISIONS = [
-	{ id: 'GY', text: 'billion years' },
-	{ id: 'MY100', text: 'hundred million years' },
-	{ id: 'MY10', text: 'ten million years' },
-	{ id: 'MY', text: 'million years' },
-	{ id: 'KY100', text: '100,000 years' },
-	{ id: 'KY10', text: '10,000 years' },
-	{ id: 'KY', text: 'millenium' },
```

Comment:
Currently these JavaScript constants are not used anywhere. They were just introduced in the unreleased 0.7.0.

PHP constants are named different, see https://gerrit.wikimedia.org/r/#/c/194306/. This is the main reason why I touched this: 1. PHP constants use lower (!) case characters. 2. PHP and JavaScript constants are named differently.

If there is a better naming scheme, I'm open for suggestions.